SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.cxx

```diff
@@ -0,0 +1,249 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#define LOCTEXT_NAMESPACE "SpatialConstants"
+
+using namespace SpatialGDK;
+
+namespace SpatialConstants
+{
+
+FString RPCTypeToString(ERPCType RPCType)
+{
+    switch (RPCType)
+    {
+    case ERPCType::ClientReliable:
+        return TEXT("Client, Reliable");
+    case ERPCType::ClientUnreliable:
+        return TEXT("Client, Unreliable");
+    case ERPCType::ServerReliable:
+        return TEXT("Server, Reliable");
+    case ERPCType::ServerUnreliable:
+        return TEXT("Server, Unreliable");
+    case ERPCType::ServerAlwaysWrite:
+        return TEXT("Server, AlwaysWrite");
+    case ERPCType::NetMulticast:
+        return TEXT("Multicast");
+    case ERPCType::CrossServer:
+        return TEXT("CrossServer");
+    }
```

Comment:
default

Reply:
though i now realise this was here already, just moved around - will see if I can lower it to a nit.